cpp-package/scripts/OpWrapperGenerator.py

```diff
@@ -88,8 +88,10 @@ class Arg:
         'caffe-layer-parameter':'::caffe::LayerParameter',\
         'NDArray-or-Symbol[]':'const std::vector<Symbol>&',\
         'float':'mx_float',\
+        'float or None':'mx_float',\
```

Comment:
Why not `'float or None':'dmlc::optional<float>',\`?

Reply:
Thank you for pointing it! I will fix it.